app/services/move_events/params_validator.rb

```diff
@@ -50,6 +51,11 @@ def initialize(move, params)
       singularize_acceptable_plural_types
     end
 
+    def to_location
```

Comment:
I think location validation should be a function of the LocationValidator. How about passing in the move type or move record to the location validator in order to do more comprehensive validations?